src/Store/Cart.js

```diff
@@ -0,0 +1,10 @@
+import { create } from "zustand";
+
+const useCartStore = create((set) => ({
+  cart: [],
+  addToCart: (product) => set((state) => ({ cart: [...state.cart, product] })),
+  removeCart: (product) =>
```

Comment:
I think this property should be called `removeFromCart` so that it's consistent with `addToCart`